<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagina_rutas</class>
 <widget class="QWidget" name="pagina_rutas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_estatico_origen">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Origen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ComboBox_origen">
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}

QComboBox:hover {
    border-color: #c0c4cc;
}

QComboBox:focus {
    border-color: #409eff;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 4px solid #909399;
}

QComboBox QAbstractItemView {
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    background-color: white;
    outline: none;
    selection-background-color: #409eff;
    selection-color: white;
    /* ELIMINA CUALQUIER cursor: ...; DE AQUÍ */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_estatico_destino">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Destino</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ComboBox_destino">
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}

QComboBox:hover {
    border-color: #c0c4cc;
}

QComboBox:focus {
    border-color: #409eff;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 4px solid #909399;
}

QComboBox QAbstractItemView {
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    background-color: white;
    outline: none;
    selection-background-color: #409eff;
    selection-color: white;
    /* ELIMINA CUALQUIER cursor: ...; DE AQUÍ */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="boton_agregar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Añadir Ruta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_cancelar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #FF6A36;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>211</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_estatica_distancia">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Distancia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_distancia">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}

QLineEdit:hover {
    border-color: #c0c4cc;
}

QLineEdit:focus {
    border-color: #409eff;
}

QLineEdit:disabled {
    background-color: #f5f7fa;
    color: #c0c4cc;
    border-color: #e4e7ed;
}

QLineEdit:read-only {
    background-color: #f5f7fa;
    color: #909399;
}</string>
      </property>
      <property name="inputMask">
       <string>00000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_estatico_titulo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Añadir nueva ruta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
